docs/documentation.rst

```diff
@@ -0,0 +1,46 @@
+Generating the documentation locally
+----------------------------------
+
+Install the required packages
+
+.. code-block:: bash
+
+   $ pip install sphinx sphinx-autobuild sphinxcontrib-napoleon
+
+.. tip::
+   If you don't have sudo privileges, you can use the ``-- user`` flag to 
+   install the required packages locally.
+
+Enter in the docs directory and rename the existing conf.py
```

Comment:
This has to be done to generate the Makefile (and the `make.bat` file for Windows users).
An alternative is to commit them, I thought you didn't do it in the first place for a reason.